lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+# A turn is initialized with two arguments.
+#1st is a string representing a guess to a card’s question.
+#2nd argument is a Card object representing the current flashcard being shown.
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+#This method returns a boolean
+#indicating if the guess matched the answer on the Card
+  def correct?
+    guess == card.answer
+  end
+
+#This method either returns "Correct!" or "Incorrect."
+#based on whether the guess was correct or not.
+  def feedback
+    return "Correct!"
+  end
+
+  def feedback2
```

Comment:
This method `feedback2` is not in the spec, and I think you were trying to have two different methods return separate feedback. However, both sets of feedback (both Correct and Incorrect) should be able to be returned from the same `feedback` method. 

What kind of syntax could you use to determine a course of action in code, based on a condition?